Cargo.toml

```diff
@@ -121,6 +121,7 @@ gsdk = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
 gclient = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
 gear-core = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
 gbuiltin-bls381 = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
+vara-runtime = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
```

Comment:
Isn't it too heavy dependency? If so, maybe just introduce `const UNITS` directly in the code as it's unlikely to be changed and also not used in critical code, only in metrics

Reply:
agree )